src/components/Accordion/Accordion.tsx

```diff
@@ -49,7 +52,8 @@ const getIcon = (isExpanded = false, index: number, id: string) => {
 const getElementsFromData = (
   data: AccordionDataProps[] = [],
   id: string,
-  panelMaxHeight: string
+  panelMaxHeight: string,
+  isAlwaysRendered: boolean = false
```

Comment:
Nit: alphabetize the list items

Reply:
Oh, these are arguments for the function and I don't think they need to be alphabetized, but I'll do it for this sake.